apps/condo/domains/user/access/User.js

```diff
@@ -187,6 +175,32 @@ const canAccessMarketingConsent = {
     update: access.userIsAdminOrIsThisItem,
 }
 
+async function userIsAdminOrIsThisItemOrCanDirectlyReadField (args) {
+    const nonDirectAccess = access.userIsAdminOrIsThisItem(args)
+
+    if (nonDirectAccess) {
+        return nonDirectAccess
+    }
+
+    const { authentication: { item: user }, listKey, fieldKey } = args
+
+    return await canDirectlyReadSchemaField(user, listKey, fieldKey)
+}
+
+async function userIsAdminOrCanDirectlyManageField ({ authentication: { item: user }, listKey, fieldKey }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    return await canDirectlyManageSchemaField(user, listKey, fieldKey)
+}
+
+const canAccessRightsSet = {
+    read: access.userIsAdminOrIsSupport,
+    create: userIsAdminOrCanDirectlyManageField,
+    update: userIsAdminOrCanDirectlyManageField,
```

Comment:
Given the sensitivity of these permissions can u please write a test case that would cover our scenario:

1. I as admin create a special set of permissions
2. No one other then admin can create this special set of permissions
3. If I have these permissions I can read email / phone ...